publication-event-handlers/src/main/java/no/unit/nva/publication/events/handlers/expandresources/ExpandResourcesHandler.java

```diff
@@ -53,10 +57,25 @@ protected EventPayload processInputPayload(DynamoEntryUpdateEvent input,
                                                Context context) {
 
         return Optional.ofNullable(input.getNewData())
+            .filter(this::shouldBeEnriched)
             .flatMap(this::transformToJson)
             .map(this::insertEventBodyToS3)
+            .stream()
+            .peek(uri -> logger.info("S3 URI:" + uri.toString()))
             .map(EventPayload::indexedEntryEvent)
-            .orElse(EventPayload.emptyEvent());
+            .collect(SingletonCollector.collectOrElse(EventPayload.emptyEvent()));
+    }
+
+    private boolean shouldBeEnriched(ResourceUpdate entry) {
```

Comment:
We do not need to do anything with Draft publications and doi requests of draft publications